content/en/docs/reference/glossary/cidr.md

```diff
@@ -0,0 +1,18 @@
+---
+title: CIDR
+id: cidr
+date: 2019-11-12
+full_link: 
+short_description: >
+  CIDR is a notation for describing blocks of IP addresses and is used heavily in various networking configurations.
+
+aka:
+tags:
+- networking
+---
+CIDR ( Classless Inter-Domain Routing) is a notation for describing blocks of IP addresses and is used heavily in various networking configurations.
+
+<!--more-->
+
+It is an IP addressing scheme that replaces the older system based on classes A, B, and C. With CIDR, a single IP address can be used to designate many unique IP addresses. A CIDR IP address looks like a normal IP address except that it ends with a slash followed by a number, called the IP network prefix.
```

Comment:
That's the correct meaning of [CIDR](https://tools.ietf.org/rfc/rfc1817.txt) generally. However, Kubernetes uses _CIDR_ to mean “a block of IP addresses that you can describe with an start address and a netmask”. For example, various IPv6 settings use the term CIDR even though CIDR itself is purely an IPv4 concept.

Reply:
To that end, maybe something add something specific to how it is used in Kubernetes, like `In Kubernetes, each node is assigned a range of IP addresses through the start address and a netmask. This allows nodes to assign each <Pod> a unique IP address.`